src/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -569,28 +554,10 @@ SHARED_API int corehost_main_with_output_buffer(const int argc, const pal::char_
 
 int corehost_libhost_init(hostpolicy_init_t &hostpolicy_init, const pal::string_t& location, arguments_t& args)
 {
-    if (trace::is_enabled())
-    {
-        trace_hostpolicy_entrypoint_invocation(location);
-        trace::info(_X("}"));
-
-        trace::info(_X("Deps file: %s"), hostpolicy_init.deps_file.c_str());
-        for (const auto& probe : hostpolicy_init.probe_paths)
-        {
-            trace::info(_X("Additional probe dir: %s"), probe.c_str());
-        }
-    }
-
```

Comment:
Where did the tracing code move to?  I didn't see it being inserted into another function.  (Here and below)

Reply:
It's in `corehost_init`. I basically split `corehost_main_init` out into that, so the diff doesn't show it as added/changed - I think that section of code is actually collapsed in the default diff view =/
In the changed file, it is at lines 460-475 and line 482.